manifests/j/JFrog/Conan/2.10.0/JFrog.Conan.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: JFrog.Conan
+PackageVersion: 2.10.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: Conan Package Manager_is1
+ReleaseDate: 2024-12-03
+AppsAndFeaturesEntries:
+- ProductCode: Conan Package Manager_is1
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '{code:DefDirRoot}\Conan'

```

Comment:
Any clue where this `DefDirRoot` variable comes from? It doesn't look right and is now seeping into newer manifests: https://github.com/microsoft/winget-pkgs/pull/202046